### PR TITLE
add accuracy limit argument

### PR DESCRIPTION
**Pull request by TomGoBravo**

### CODE REVIEW

File: extract [hunk 1]
```diff
@@ -45,6 +45,7 @@ parser.add_argument('end_year_month', nargs='?')
 
 parser.add_argument('-v', '--verbose', action='store_true')
 parser.add_argument('-o', '--output-dir', default='output')
+parser.add_argument('--accuracy-limit', type=int, default=None)
```
Comment (andrewkww) [inline extract]: When someone says "limit", I tend to think of a maximum, like "speed limit". In this case, the flag is actually specifying a minimum. Would something like `--min-accuracy` be more concise?

Comment (TomGoBravo) [reply extract]: Does 64d2ab0 help?

Comment (andrewkww) [reply extract]: Ah, I see now. Yes, that makes sense.

**Comment by andrewkww** [comment]
Thanks!

**Comment by TomGoBravo** [comment]
Thank you for saving me from writing a location history converter and quickly merging my little change!

**Pull request merged**